### DISCUSSION TITLE
How to pass metadata to a file ?

### DISCUSSION THREAD

**Discussion by karol-unum-la** [uploadcare/file-uploader#169] [General]
Hello, 

I am trying to figure out how to pass metadata to file before an upload.  I know that Upload Care API supports such a possibility. However is it possible using the uc-blocks ? Can you explain to me how to achieve it?

**Comment by nd0ut** [comment]
dup #170, fixed in #172

sorry I didn't see it before you emailed me :)

**Comment by kkerti** [comment]
It would be cool to mention in the docs that `ctx-name` should be set on `<lr-upload-ctx-provider/>` to match with the `<lr-file-uploader-inline/>` config to make it work. Else there is a symbiote error:

`Symbiote Data: cannot publish. Prop name: uploadMetadata`

```
<lr-file-uploader-inline 
      css-src="https://unpkg.com/@uploadcare/blocks@0.14.1/web/file-uploader-inline.min.css"
      class="uploader-cfg"
    >
      <lr-upload-ctx-provider 
        ctx-name="SAME AS THE LR-FILE-UPLOADER-INLINE CTX-NAME" 
        bind:this={uploader}
      >
      </lr-upload-ctx-provider>
</lr-file-uploader-inline>
```
